app/helpers/user_helper.rb

```diff
@@ -0,0 +1,10 @@
+module UserHelper 
```

Comment:
in rails the helpers (in theory) are used by the views. So either put the `subscribe(user)` in the user controller or create a concern for it.

In the case like it is only one method, I would suggest to just move to the  user controller

Reply:
i will just update the register_for_announce_list to handle many subscribe service , so no need to create mailer

Reply:
OK for now it will be enough but in the future, if we have multiple services your first implementation was better.